docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -80,6 +80,20 @@ For example, you might opt to run and interact with the GX Core Python library l
 
 GX provides flexible, robust products that allow your organization to quickly deploy GX Cloud or GX Core to fit your unique requirements.
 
+## Credential management 
```

Comment:
The About GX page is currently being overhauled (by me) in a separate PR and won't be a suitable place for this information after the reorganization is complete. I'd suggest we add this information to the *Manage Data Assets* page instead, since it's relevant to Data Source creation.

Reply:
Looking at the content in https://github.com/great-expectations/great_expectations/pull/10417/files, I think it'd be helpful to combine the two together into a _Data Source credential management_ section (within the _Manage Data Assets_ page) that describes both Direct input and Environment variable substitution - that way, all the relevant information is in one place.